content/open-source/concepts/graphql/graphql.md

```diff
@@ -25,9 +25,11 @@ GraphQL was initially developed by Meta (formerly Facebook) in 2012 to address i
 
 Before adopting GraphQL, developers should be familiar with RESTful architecture.
 
-- One of REST's disadvantages is it can result in an excessive number of [HTTP](https://www.codecademy.com/resources/docs/general/http) queries or too much data being fetched.
-- Alternatively, GraphQL allows developers to ask for specific data instead of fetching a large object with all of the data every time. There is no room for errors like underfetching and overfetching with GraphQL.
-- Finally, GraphQL allows developers to easily integrate with [databases](https://www.codecademy.com/resources/docs/general/database), [JSON](https://www.codecademy.com/resources/docs/general/json) files, RESTful APIs, and cloud services.
+- GraphQL allows developers to easily integrate with [databases](https://www.codecademy.com/resources/docs/general/database), [JSON](https://www.codecademy.com/resources/docs/general/json) files, RESTful APIs, and cloud services.
+- It prioritises giving clients the exact data that they require and nothing more, so that we only receive the data that we need, helping to eliminate over-fetching.
+- It can be used to construct requests that pull data from multiple sources into a single API call and response.
+- Easier to adopt new data to existing objects, helping to improve future changes and potential under-fetching.
+  - For example - We have an existing setup with objects and types already defined but we now need to include more data in our object, this could require a large change of including an additional fetch within our flow, or include additional steps when modelling/mapping the responses to our object.
```

Comment:
Can we remove any pronouns (e.g., we and you) and adjust accordingly for these points in addition to the rest of the additions in this entry?